dev/breeze/src/airflow_breeze/commands/testing_commands.py

```diff
@@ -185,6 +192,65 @@ def run_with_progress(
     finally:
         os.unlink(f.name)
     return result
+    
+def run_tests(
+    env_variables: Dict[str, str],
+    dry_run: bool, 
+    verbose: bool, 
+    test_type: str,
+    extra_pytest_args: Tuple
+) -> Tuple[int, str]:
+    if test_type:
+        env_variables['TEST_TYPE'] = test_type
+    perform_environment_checks(verbose=verbose)
+    cmd = ['docker-compose', 'run', '--service-ports', '--rm', 'airflow']
+    cmd.extend(list(extra_pytest_args))
+    test_result = run_command(
+        cmd,
+        verbose=verbose,
+        dry_run=dry_run,
+        env=env_variables,
+    )
+    return (
+        test_result.returncode,
+        f"Running tests {test_type}",
+    )
+
+
```

Comment:
@potiuk Share your review in this code when you get time. I didn't yet add the parser part. But your reviews until this part would be useful. Thanks